src/mir/index.md

```diff
@@ -44,7 +44,8 @@ This section introduces the key concepts of MIR, summarized here:
   leading underscore, like `_1`. There is also a special "local"
   (`_0`) allocated to store the return value.
 - **Places:** expressions that identify a location in memory, like `_1` or
-  `_1.f`.
+  `_1.f`. (Some people call these "lvalues"; the "l" meaning that they occur on
+  the left-hand side of an assignment.)
```

Comment:
This is traditional in C (especially in compiler textbooks) but it's not correct in Rust, because they can be on the right hand side. For example `&mut MyTemporary{}` is a place expression that can be used in an expression context.

Reply:
Hmm, okay.

Reply:
The only place in the whole compiler that uses this is https://github.com/rust-lang/rust/blob/9e5f7d5631b8f4009ac1c693e585d4b7108d4275/compiler/rustc_mir/src/transform/simplify.rs#L463
Maybe it should be changed?